data/migrations/20190708170952_create_receipts_table.js

```diff
@@ -0,0 +1,19 @@
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('receipt', tbl => {
+      tbl.increments();
+      tbl.string('receipt_name', 128);
+      tbl.string('receipt_image_url', 500)
+      tbl
+  
+        .integer('user_id')
+        .unsigned()
+        .references('uid')
+        .inTable('users')
+        .onDelete('CASCADE')
+        .onUpdate('CASCADE'); // links to user table/owner of receipt
+    });
+  };
+  
+  exports.down = function(knex, Promise) {
+    return knex.schema.dropTableIfExists('property_image');
```

Comment:
Same issue as https://github.com/labs13-property-mgr/backend/pull/51 , I think this should be changed to 'receipt' instead of 'property_image'.

Reply:
fixed it